public/index.html

```diff
@@ -6,6 +6,17 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<!-- <link rel="icon" type="image/png" href="/images/favicon.png" /> -->
 		<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+		<style>
```

Comment:
Added this Style tag to have these few base styles apply to the whole project now that I have added Radium, I figured this was the best place.